src/PIL/DdsImagePlugin.py

```diff
@@ -157,6 +159,10 @@ def _open(self):
                     self.pixel_format = "BC7"
                     self.info["gamma"] = 1 / 2.2
                     n = 7
+                elif dxgi_format in (DXGI_FORMAT_R8G8B8A8_TYPELESS,
+                                     DXGI_FORMAT_R8G8B8A8_UNORM):
```

Comment:
```suggestion
                elif dxgi_format in (
                    DXGI_FORMAT_R8G8B8A8_TYPELESS,
                    DXGI_FORMAT_R8G8B8A8_UNORM,
                ):```

This will ix the failing lint test.